tools/tagging/changes.sh

```diff
@@ -1,11 +1,13 @@
 #!/bin/bash
 #Display a list of the last changes for each repo for a given version to help decide whether anything needs upmerging
 
+gitroot="${gitroot/#\~/$HOME}"
```

Comment:
I suspect that is always fine, could use 'realpath' instead to potentially cover other cases ?
gitroot="$(realpath $gitroot)"